regolith/helpers/l_contactshelper.py

```diff
@@ -23,10 +21,8 @@
 
 def subparser(subpi):
     subpi.add_argument(
-        "-u",
-        "--run",
-        required=True,
-        help='required argument: "y" must be entered here in order for the lister to run')    
+        "run",
+        help='required argument that must be entered here in order for the contacts lister to run')
```

Comment:
how about ``"run the lister.  To see allowed optional arguments type regolith helper l_contacts"``